default.vcl

```diff
@@ -23,10 +23,10 @@ sub vcl_recv {
     elif (req.url ~ "^\/content\/.*$") {
         set req.backend_hint = dynBackend.backend("content-public-read");
     }
-    elif (req.url ~ "^\/enrichedcontent\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$") {
+    elif (req.url ~ "^\/enrichedcontent\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}\?.*$") {
```

Comment:
Doesn't make this change _required_ to have a query param? Is the query param required for this and for the below endpoints?